### ISSUE TITLE
Not equals in table row does not align correctly [pen included]

### ISSUE THREAD

**Issue by HughGrovesArup** [KaTeX/KaTeX#2100]
https://codepen.io/HughGrovesArup/pen/OJLadYp

As can be seen when in a table row the / is misaligned.

![image](https://user-images.githubusercontent.com/40033911/65340273-cca64600-dbc5-11e9-91a5-ffdb60aa29fa.png)

**Comment by kevinbarabash** [comment]
The issue doesn't appear on https://katex.org/.  My hunch is that codepen.io's CSS is affect ours somehow.  I wasn't able to figure out what style was responsible though. :(  Once we figure that out we should be able to guard against it.  We should probably add a bunch of reset params to `.katex` in katex.less to reset all external styles.

**Comment by HughGrovesArup** [comment]
The places where I have actually encountered the issue are in a VS code extension that utilises a webview (like an iframe) and inside a vue.js component.

The issue does not appear on katex.org - but is that just because it's not within <tr><td> tags? If I remove the tr / td from the codepen it renders as expected https://codepen.io/HughGrovesArup/pen/LYPMvxW

**Comment by mbourne** [comment]
@HughGrovesArup: Removing the td and tr means the math block is removed from the table, so that's why it works.

I did some testing.

(1) Removing ALL of codepen's styles gave the following (with the second set of equations being in p tags, not in a table):
![image](https://user-images.githubusercontent.com/7541940/65410645-821f0680-de1d-11e9-9289-93dc4daf0d32.png)

So that's not it.

(2) I then tried the code in various versions:

0.9.0 (not quite aligned, but at least looks OK)
![image](https://user-images.githubusercontent.com/7541940/65410928-26a14880-de1e-11e9-9837-41e6428023fc.png)

0.10.0 
![image](https://user-images.githubusercontent.com/7541940/65410896-0f625b00-de1e-11e9-9df0-48aaceb75694.png)

0.10.2
![image](https://user-images.githubusercontent.com/7541940/65411049-6e27d480-de1e-11e9-95ba-2ad4e1e1a9e2.png)
